foundation/okfplugins/image/migrations/0007_okimage_in_carousel.py

```diff
@@ -4,15 +4,14 @@
 
 
 class Migration(migrations.Migration):
-
     dependencies = [
-        ('image', '0006_okimage_full_width'),
+        ("image", "0006_okimage_full_width"),
```

Comment:
If there are some kind of python formatter enabled, I'll prefer to be disable for `**/migrations/*.py`
There are a lot of files changed that won't fail but changes the past.